tests/SelfTest/UsageTests/Condition.tests.cpp

```diff
@@ -223,12 +225,12 @@ TEST_CASE( "Comparisons with int literals don't warn when mixing signed/ unsigne
 // Disable warnings about sign conversions for the next two tests
 // (as we are deliberately invoking them)
 // - Currently only disabled for GCC/ LLVM. Should add VC++ too
-#ifdef  __GNUC__
```

Comment:
This line means if compiler is `GNUC` or `clang`.
But I just replaced it by `CATCH_COMPILER_GCC` cuases compiler warning -> error.
I'm searching for this situations in code.